pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -56,6 +57,8 @@
 
 assert lib.assertMsg (!(sanitizeAddress && sanitizeThreads))
   "'sanitizeAddress' and 'sanitizeThreads' are mutually exclusive, use one.";
+assert testing -> !stable
+  -> throw "testing implies stable and cannot be used with stable = false";
```

Comment:
@pbsds i test for testing / stable mutual exclusivity here
fairly sure i confirmed this works as intended